WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -159,9 +160,17 @@ args switch
 
 		WalletCoinsCommand = ReactiveCommand.Create(() => Navigate(NavigationTarget.DialogScreen).To(new WalletCoinsViewModel(this, balanceChanged)));
 
+		CoinJoinSettingsCommand = ReactiveCommand.Create(() => Navigate(NavigationTarget.DialogScreen).To(CoinJoinSettings), Observable.Return(!wallet.KeyManager.IsWatchOnly));
+
 		CoinJoinStateViewModel = new CoinJoinStateViewModel(this, balanceChanged);
+
+		IsWatchOnly = wallet.KeyManager.IsWatchOnly;
 	}
 
+	public CoinJoinSettingsViewModel CoinJoinSettings { get; }
+
+	public bool IsWatchOnly { get; }
```

Comment:
ctor would be a tiny bit shorter.

```suggestion
	public bool IsWatchOnly => Wallet.KeyManager.IsWatchOnly;
```